The AppVeyor Windows CI machine [fails](https://ci.appveyor.com/project/avsm/ocaml/build/1.0.1427) with the following error:

```
cl -nologo -D_CRT_SECURE_NO_DEPRECATE -I../../byterun -O2 -Gy- -MD -Ox \
  /Fo st_stubs_b.obj -c st_stubs.c
cl : Command line warning D9027 : source file 'st_stubs_b.obj' ignored
st_stubs.c
../../boot/ocamlrun ../../tools/ocamlmklib -o threads st_stubs_b.obj 
** Fatal error: Cannot find file "st_stubs_b.obj"
make[1]: *** [Makefile:59: libthreads.lib] Error 2
```

Doesn't that mean that Windows is in fact negatively affected by the change?

--- a/otherlibs/systhreads/Makefile
+++ b/otherlibs/systhreads/Makefile
@@ -92,7 +92,7 @@ $(LIBNAME).cmxa: $(THREADS_NCOBJS)
 ifeq "$(TOOLCHAIN)" "msvc"
   CCOUTPUT=/Fo
 else
-  CCOUTPUT=-o
+  CCOUTPUT=-o 
 endif
 
 st_stubs_b.$(O): st_stubs.c $(HEADER)

